cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/track-popover-control.tsx

```diff
@@ -0,0 +1,173 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+
+import {
+    Row,
+    Col,
+    Select,
+    InputNumber,
+    Tooltip,
+} from 'antd';
+
+import Text from 'antd/lib/typography/Text';
+
+import { TrackerInputs } from 'reducers/interfaces';
+
+interface Props {
+    tracker_type: string | null;
+    tracker_until: string | null;
+    tracker_frame_number: number;
+
+    handleChange(name: string, value: string | number): void;
+}
+
+const trackTypeOptions = [
+    {
+        label: "OpenCV Boosting",
+        value: "Boosting",
+    },
+    {
+        label: "OpenCV MIL",
+        value: "MIL",
+    },
+    {
+        label: "OpenCV KCF",
+        value: "KCF",
+    },
+    {
+        label: "OpenCV CSRT",
+        value: "CSRT",
+    },
+    {
+        label: "OpenCV MedianFlow",
+        value: "MEDIANFLOW",
+    },
+    {
+        label: "OpenCV TLD",
+        value: "TLD",
+    },
+    {
+        label: "OpenCV MOSSE",
+        value: "MOSSE",
+    },
+    {
+        label: "GOTURN",
+        value: "GOTURN",
+    },
+]
+
+const trackUntilOptions = [
+    {
+        label: "Next keyframe",
+        value: "nextKeyframe"
+    },
+    {
+        label: "Next manual keyframe",
+        value: "nextUserKeyframe"
+    },
+    {
+        label: "Next manual keyframe (at most 2)",
+        value: "nextUserKeyframeMax2",
+    },
+    {
+        label: "Next manual keyframe (at most 5)",
+        value: "nextUserKeyframeMax5"
+    },
+    {
+        label: "Next manual keyframe [at most 10]",
+        value: "nextUserKeyframeMax10"
+    },
+    {
+        label: "Next manual keyframe [at most 15]",
+        value: "nextUserKeyframeMax15"
+    },
+    {
+        label: "Next manual keyframe [at most 20]",
+        value: "nextUserKeyframeMax20"
+    },
+    {
+        label: "Next manual keyframe [at most 30]",
+        value: "nextUserKeyframeMax30"
+    },
+    {
+        label: "Next manual keyframe [at most 50]",
+        value: "nextUserKeyframeMax50"
+    },
+    {
+        label: "End",
+        value: "end"
+    },
+]
+
+function TrackPopoverComponent(props: Props) : JSX.Element {
+    const {
+        tracker_type,
```

Comment:
Please, use camelCase syntax instead of ``_`` where it is possible

Reply:
Corrected